docs/source/nodes_and_pipelines/namespaces.md

```diff
@@ -164,3 +182,55 @@ def create_pipeline(**kwargs) -> Pipeline:
 After executing the pipeline with `kedro run`, the visualisation with `kedro viz run` after collapsing will look like this:
 
 ![namespaces collapsed all](../meta/images/namespaces_collapsed_all.png)
+
+## Group nodes with namespaces
+
+You can [namespace an entire pipeline](#what-is-a-namespace) to enhance visualisation in Kedro-Viz if your pipeline structure aligns with the desired node grouping. But if you want to group specific nodes rather than an entire pipeline, you can do so by assigning the `namespace` argument in the node creation function. In this case, only the node name will be prefixed with `namespace_name.`, while inputs, outputs, and parameters will remain unchanged. Note that this behavior differs from [namespacing at the pipeline level](#what-is-a-namespace).
```

Comment:
Adding the link is a good idea 👍 Just one more change for UK spelling.

```suggestion
You can [namespace an entire pipeline](#what-is-a-namespace) to enhance visualisation in Kedro-Viz if your pipeline structure aligns with the desired node grouping. But if you want to group specific nodes rather than an entire pipeline, you can do so by assigning the `namespace` argument in the node creation function. In this case, only the node name will be prefixed with `namespace_name.`, while inputs, outputs, and parameters will remain unchanged. Note that this behaviour differs from [namespacing at the pipeline level](#what-is-a-namespace).
```